external-resolver/src/test/scala/org/genivi/sota/resolver/test/random/Misc.scala

```diff
@@ -20,6 +20,10 @@ object Misc {
   def lift[X, S](gen: Gen[X]): StateT[Gen, S, X] =
     StateT(s => gen.flatMap(x => (s, x)))
 
+  def lift2[S, A](blk: => A): cats.state.State[S, A] =
```

Comment:
Is this used? If so how? Maybe another name would be appropriate; `lift` is taken from Haskell...